content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx

```diff
@@ -542,7 +541,7 @@ const checkJson = await prisma.user.findMany({
 - If you are using PostgreSQL, you must pass in an array of objects to match, even if that array only contains one object:
 
   ```js
-  [{ "status": "expired", "insuranceID": 92 }] // PostgreSQL
+  ;[{ status: 'expired', insuranceID: 92 }] // PostgreSQL
```

Comment:
What's up with the leading `;` weird choice, prettier.

Reply:
This is because of this rule: https://github.com/prisma/docs/blob/fad88c6f110219f4d2f567961036efd01e7d4704/.prettierrc#L7

Here is some context why prettier adds this https://github.com/prettier/prettier/issues/1935#issuecomment-306729468